cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -93,7 +92,7 @@ fn define_control_flow(
         "#,
                 &formats.branch_table,
             )
-            .operands_in(vec![x, label, JT])
+            .operands_in(vec![x, JT, block_call])
```

Comment:
Is there some limitation in the code generator that requires this change?

Reply:
Yes, with the introduction of `BlockCall`, block arguments got floated to the end of instruction builders. I think that's fixable, but in the interest of getting something building I switched it locally. I'll add a TODO to investigate fixing this.